config/openvpn-client-export/openvpn-client-export.inc

```diff
@@ -44,20 +44,17 @@ $current_openvpn_version_rev = "03";
 
 function openvpn_client_export_install() {
 	global $current_openvpn_version;
+	conf_mount_rw();
+	$tarpath = "/usr/local/pkg/openvpn-client-export-{$current_openvpn_version}.tgz";
+	$phpfile = "vpn_openvpn_export.php";
```

Comment:
I have not looked hard, but why does this line need to be added?
And why do you need to take all this code here out of the $pfs_version tests - maybe so it works for 2.3 and onwards?